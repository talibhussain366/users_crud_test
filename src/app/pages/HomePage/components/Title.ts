import styled from 'styled-components/macro';

export const Title = styled.div`
  color: white;
  font-size: 15px;
`;
