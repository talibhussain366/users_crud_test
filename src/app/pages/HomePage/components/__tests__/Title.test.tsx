import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Title } from '../Title';

const shallowRenderer = createRenderer();

describe('<Title />', () => {
  it('<Title />', () => {
    shallowRenderer.render(<Title />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
