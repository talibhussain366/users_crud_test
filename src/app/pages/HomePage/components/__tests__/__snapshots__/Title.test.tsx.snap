// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> <Title /> 1`] = `
<div
  className="Title-sc-rvokxi-0 cZbDwV"
/>
`;
